pkg/tfvars/internal/cache/cache.go

```diff
@@ -156,7 +161,7 @@ func DownloadFile(baseURL string, dataType string) (string, error) {
 	_, err = os.Stat(filePath)
 	if err == nil {
 		logrus.Infof("The file was found in cache: %v. Reusing...", filePath)
-		return filePath, nil
+		return filePath, decompressFile(filePath, filePath)
```

Comment:
multiple people could actually start decompressing this file? while the cacheFile only allows one person to do the cache-modifications...

Reply:
Probably do need an flock in there.

Reply:
Fixing.

Reply:
Obsolete.